website/docs/components/table/advanced-table/partials/accessibility/accessibility.md

```diff
@@ -14,6 +14,78 @@ The table row element cannot receive interactions, meaning actions cannot be att
 
+## Keyboard Navigation
+
+The Advanced Table supports two keyboard interaction modes: Navigation Mode and Action Mode. These modes allow users to move through table data and interact with elements using a keyboard.
+
+### Navigation Mode
+
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+When a user tabs into the Advanced Table, the first cell is focused.
+
+![Keyboard focus on the sortable 'Artist' column header with sorting controls active.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-tab.png =402x*)
+
+<Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" @title="Left arrowy" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" @title="Right arrow" /></Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up" @title="Up arrow" /></Doc::Badge> <Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down" @title="Down arrow" /></Doc::Badge>
+
+Arrow keys move between cells. If the table extends beyond the screen, arrow keys allow movement to access hidden columns.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-arrow-key-down.png =402x*)
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-left" @title="Left arrow" /></Doc::Badge>
+
+Move focus to the first cell in the row.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-left.png =402x*)
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-right" @title="Right arrow" /></Doc::Badge>
+
+Move focus to the last cell in the row.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-right.png =402x*)
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-up"  @title="Up arrow" /></Doc::Badge>
+
+Move focus to the first cell in the column.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-up.png =402x*)
+
+<Doc::Badge @type="neutral">Fn</Doc::Badge><Doc::Badge @type="neutral"><Hds::Icon @name="arrow-down"  @title="Down arrow" /></Doc::Badge>
+
+Move focus to the last cell in the column.
+
+![Keyboard focus on 'The Beatles' row using the arrow key.](/assets/components/table/advanced-table/advanced-table-keyboard-navigation-fn-down.png =402x*)
+
+### Action Mode
+
+<Doc::Badge @type="neutral">Enter</Doc::Badge>
+
+Press Enter in Navigation Mode to enter Action Mode and interact with elements inside a cell. 
+
+- If the cell contains one interactive element, focus moves to that element instead of activating it.
+- If multiple interactive elements exist, the focus moves to the first one. Use the tab key to navigate between them.
```

Comment:
Focus will always go to the first interactive element, so this can be slightly streamlined. You also mentioned tabs moving between elements below, so I removed that. 

```suggestion
Press Enter in Navigation Mode to enable Action Mode and interact with elements inside a cell. Focus will move to the first actionable element.
```